cmd/openapi-gen/builder/openapi.go

```diff
@@ -285,19 +294,19 @@ func (o *openAPI) buildResponses(handler interface{}, results []definition.Resul
 			// TODO(liubog2008):
 			haveError = true
 		case definition.Meta:
-			for _, k := range res.Headers {
-				rightResponse.Headers[k] = spec.Header{
-					// TODO(liubog2008): now only string is supported
-					SimpleSchema: spec.SimpleSchema{
-						Type:   "string",
-						Format: "",
-					},
-					// TODO(liubog2008): add header description for each one
-					HeaderProps: spec.HeaderProps{
-						Description: res.Description,
-					},
-				}
-			}
+			// for _, k := range res.Headers {
```

Comment:
why comment out

Reply:
I have discussed is with @liubog2008. The field is not graceful here. So I delete the field and comment out related codes.
We'll continue to find out the best practice of the document in definitions.